aerotech_101smc2/aerotech_101smc2/_aerotech.py

```diff
@@ -7,7 +7,7 @@
 
 
 class Aerotech(ContinuousHardware):
-    _kind = "my-daemon"
+    _kind = "Aerotech_101smc2"
```

Comment:
```suggestion
    _kind = "aerotech-101smc2"
```

the standard here is all lower case, dashes between words